components/Modal/CreateImageUpload.js

```diff
@@ -123,7 +123,11 @@ class CreateImageUploadModal extends React.Component {
   }
 
   setUploadService(_, event) {
-    this.setState({ uploadService: event.target.value });
+    if (this.state.uploadService === event.target.value) {
```

Comment:
Please no fix-up commits. This should be merged with the previous commit where this functionality was introduced. In general, each commit in the PR should not introduce unfinished/problematic code waiting for followup commits to fix the issues.

Reply:
Noted and fixed